src/device/usbd.c

```diff
@@ -708,17 +722,17 @@ static bool process_set_config(uint8_t rhport, uint8_t cfg_num)
     uint8_t drv_id;
     uint16_t drv_len;
 
-    for (drv_id = 0; drv_id < USBD_CLASS_DRIVER_COUNT; drv_id++)
+    for (drv_id = 0; drv_id < TOTAL_DRIVER_COUNT; drv_id++)
```

Comment:
reverse order to run from TOTAL_DRIVER_COUNT-1 to 0 will effectively allow application to overwrite built-in class driver. However, I cannot think of a good reason why application would do such a thing. Like providing its own msc/cdc instead of bultin one. If there is a bug within stack driver, it should be fixed rather. So we could just leave it as it is here until there is a scenario to justify the change.

Reply:
I think an alternative when we need to override a driver is to exclude one of tinyusb driver files during the build process, and provide a custom driver implementation with the same named functions. So I don't mind if this stays as is.